.github/workflows/default.yml

```diff
@@ -10,8 +10,10 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - name: checkout submodules
-      run: git submodule update --init --recursive    
-    - name: setup
+      run: git submodule update --init --recursive
```

Comment:
~~The workflow currently fails, is this by 'design'?~~

No it does not fail XD